assets/scss/_nav.scss

```
@@ -18,7 +18,7 @@
 }
 
 #main-header {
-    .navbar {
+    .navbar-light {
```

there's no more class .navbar ?

---

```suggestion
    .navbar, .navbar-light {
```

In order to avoid breaking stuff

---

Good idea, implemented it